pep-0004.txt

```diff
@@ -280,6 +280,12 @@ Deprecated modules
     Date:          2013-08-12
     Documentation: Deprecated as of Python 3.4.
 
+    Module name:   macpath
+    Rationale:     Obsolete macpath module dangerously broken
```

Comment:
To go along with other entries, the rationale could just state "Platform is obsolete/unsupported", otherwise LGTM